src/components/Card.js

```diff
@@ -43,14 +97,23 @@ export default class Card {
     getView() {
         this._element = this._getTemplate();
         
+        this._modal = document.querySelector('.modal');
+        this._profileImage = document.querySelector('.profile__image');
+        this._profileImageButton = document.querySelector('.profile__image-edit');
```

Comment:
The avatar should be handled by the instance of the avatar modal using `PopupWithForm`